src/components/PluginManager.svelte

```diff
@@ -121,6 +122,11 @@
 	(async () => plugins = await (await fetch("https://ninjadev64.github.io/openaction-plugins/catalogue.json")).json())();
 
 	let search: string = "";
+
+	onOpenUrl((urls: any) => {
+		let id = urls[0].split("installPlugin/")[1];
+		installPluginGitHub(id, plugins[id]);
+	});
```

Comment:
Please could you add checks for when the string does not contain installPlugin or the plugin ID is not present in the plugins object, and return doing nothing when this is the case